nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java

```diff
@@ -53,19 +50,44 @@
 
     private static final Log log = LogFactory.getLog(BulkServiceImpl.class);
 
+    public static final String BULK_LOG_MANAGER_NAME = "bulk";
+
+    public static final String BULK_KV_STORE_NAME = "bulk";
+
+    public static final String COMMAND_STREAM = "command";
+
+    public static final String STATUS_STREAM = "status";
+
+    public static final String DONE_STREAM = "done";
+
+    public static final String RECORD_CODEC = "avro";
+
     public static final String COMMAND_SUFFIX = ":command";
 
     public static final String STATUS_SUFFIX = ":status";
 
+    public static final String PRODUCE_IMMEDIATE_OPTION = "produceImmediate";
+
     @Override
     public String submit(BulkCommand command) {
         if (log.isDebugEnabled()) {
             log.debug("Run action with command=" + command);
         }
         // check command
+        // TODO: check must be done on builder
```

Comment:
When will you do the TODOs?

Reply:
Soon, providing clean builder pattern for command and status rest endpoint is a priority for the current sprint